src/frontend/src/routes/(app)/+layout.svelte

```diff
@@ -45,7 +45,7 @@
 		<SplitPane>
 			<NavigationMenu slot="menu">
 				{#if route === 'tokens'}
-					<div in:fade class="hidden xl:block">
+					<div in:fade out:fade class="hidden xl:block">
```

Comment:
I think you can summarize `in:` and `out:` as `transition:` (https://svelte.dev/docs/svelte/in-and-out)

Reply:
Nice one!